developer/decision-records/2024-05027-maturity-levels-deprecation-policy/README.md

```diff
@@ -0,0 +1,76 @@
+# EDC Maturity Levels and Deprecation Policy
+
+## Decision
+
+The EDC project will adopt the API Maturity Levels and Deprecation Policy (MLDP) specified in this decision record.
+
+## Rationale
+
+Defining API maturity levels and a deprecation policy will better enable EDC users and downstream projects to plan when to adopt and migrate from specific EDC API versions. The rules and maturity levels are inspired by the [Kubernetes Deprecation Policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/)
+
+## Impact
+
+The MLDP will dictate the lifecycle of all EDC APIs. However, it will not alter EDC's milestone-based release process nor will it introduce the notion of a Long-Term Support (LTS) version. Rather, the MLDP is designed to allow the EDC project to continue with its frequent release schedule while improving planning opportunities for downstream projects. Note that these rules do not apply to EDC SPIs.
+
+## Maturity Level
+
+As detailed in the [Consistent Versioning of External APIs Decision Record](https://github.com/eclipse-edc/docs/tree/main/developer/decision-records/2024-05-20-api-versioning-2), EDC APIs are organized into individually versioned groups or "contexts." Each group version will have one of the following assigned maturity levels:
+
+- ***Alpha***: The API is experimental, not extensively tested, and may be changed without notice. 
```

Comment:
and may be changed without _prior deprecation_ notice

Reply:
I worded it this way on purpose because we will not create descriptions of changes between releases for Alpha changes. Adding "prior deprecation" implies that we will issue some type of change Alpha description for each new release.